### ISSUE TITLE
repositories information on pulp3

### ISSUE THREAD

**Issue by waldirio** [pulp/pulpcore#2322]
**Is your feature request related to a problem? Please describe.**
- Currently, when doing a troubleshooting steps, it's hard to say all the files that are part of a repo or part of a CV/CCV, which could be a lot of repos in a single place.
- Also, it should be interesting print the # of packages in the metadata and the reference for the filesystem, in this case, we can guarantee the metadata is matching with the information that should be in the filesystem.
- An option for search by specific points will be also very welcome, for example:
  - by repo name
  - by cv/ccv name
  - by filename
  - by artifact
  - all the information (maybe, this will be the standard output and/or when the customer skip the flags)

**Describe the solution you'd like**
some simple command that will return the information that we are looking for.

**Describe alternatives you've considered**
This [script](https://github.com/waldirio/satellite_tricks/)

**Comment by ggainey** [comment]
CV/CVV are katello concepts - Pulp3 doesn't know about them. After some IRC discussion, sounds more like what we'd want there, is to be able to specify a search based on Distribution-base_path (maybe a regex thereon?) So a user who has been careful about their Distribution naming/pathing rules, could do something like "show me all the repo info for everything with 'RHEL8' in the path"

@waldirio - does the above capture what we were discussing?

We should review the output of the linked script and use it as a basis for a more-formal requirements-doc.

**Comment by waldirio** [comment]
Hello @ggainey 

Yes, that sounds like a good plan for me! 

Thank you
Waldirio

**Comment by sayan3296** [comment]
Related to "CV/CVV are katello concepts" part, How about making use of the backend_identifier of the repo here as a search parameter. 

The `katello_repositories` table from foreman db gives us that which is unique for every combination of Repo CV and LCE and perhaps Pulp can easily use that to detect the distribution path and the list stuff from there.

**Comment by sayan3296** [comment]
> Related to "CV/CVV are katello concepts" part, How about making use of the backend_identifier of the repo here as a search parameter.
> 
> The `katello_repositories` table from foreman db gives us that which is unique for every combination of Repo CV and LCE and perhaps Pulp can easily use that to detect the distribution path and the list stuff from there.
> 
> -- Sayan

Obviously, we will find that out manually and pass with the script\command\tool as an argument .. :)
